more_itertools/more.pyi

```diff
@@ -148,8 +152,8 @@ def side_effect(
     after: Callable[[], object] | None = ...,
 ) -> Iterator[_T]: ...
 def sliced(
-    seq: Sequence[_T], n: int, strict: bool = ...
-) -> Iterator[Sequence[_T]]: ...
+    seq: SupportsSlicing[_T], n: int, strict: bool = ...
+) -> Iterator[_T]: ...
```

Comment:
I'm confused as to why this is returning `Iterator[_T]` and not an iterator over things that themselves support slicing?

Reply:
That is precisely what this is doing. `sliced` is a generic function that accepts any type that supports slicing and returns an iterator over that type.